src/components/Catalyst/Step1.js

```diff
@@ -0,0 +1,90 @@
+// @flow
+
+/**
+ * Step 1 for the Catalyst registration - landing
+ */
+
+import React, {useEffect} from 'react'
+import {View, SafeAreaView, Image} from 'react-native'
+import {injectIntl, defineMessages} from 'react-intl'
+import {connect} from 'react-redux'
+
+import {generateVotingKeys} from '../../actions/voting'
+import {fetchUTXOs} from '../../actions/utxo'
+import {Text, Button, ProgressStep} from '../UiKit'
+import {withTitle} from '../../utils/renderUtils'
+import {CATALYST_ROUTES} from '../../RoutesList'
+import globalMessages, {confirmationMessages} from '../../i18n/global-messages'
+import AppDownload from '../../assets/img/pic-catalyst-step1.png'
+import playstoreBadge from '../../assets/img/google-play-badge.png'
+import appstoreBadge from '../../assets/img/app-store-badge.png'
+
+import styles from './styles/Step1.style'
+
+import type {ComponentType} from 'react'
+import type {IntlShape} from 'react-intl'
+
+import type {Navigation} from '../../types/navigation'
+
+const messages = defineMessages({
+  subTitle: {
+    id: 'components.catalyst.step1.subTitle',
+    defaultMessage:
+      '!!!Before you begin, make sure to download the Catalyst Voting App.',
+  },
+})
+
+const Step1 = ({intl, generateVotingKeys, navigation, fetchUTXOs}) => {
```

Comment:
Please add types for all component props. Historically we've been lazy here and sometimes only typed external props, without including all the props that end up injected in the component (in particular, those that come from selectors).